bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants

cmsbuild opened the pull request:
No need to have separate cfi files for HcalTopology producer for LHC and SLHC mode. 
Automatically ported from CMSSW_7_6_X #11767 (original by @bsunanda).

heppye committed:
Rename Create Bd_JpsiKPi.dec to Bd_JpsiKPi.dec

VinInn committed:
cleaning up plugins

VinInn committed:
more private

VinInn committed:
more private

VinInn committed:
new outlier rejection logic

VinInn committed:
clean and move to description

VinInn committed:
improve config

VinInn committed:
dont reject outliers for merged duplicate

VinInn committed:
improve printout

VinInn committed:
Merged TrackFitters76 from repository VinInn

VinInn committed:
fix python configs

VinInn committed:
shorten track if too many outliers

VinInn committed:
add lastvalid to list of outliers

VinInn committed:
add comment about lastvalid to list of outliers

VinInn committed:
Merged TrackFitters76 from repository VinInn

VinInn committed:
try to take into account slit matched-hit

VinInn committed:
add number of hits in seed to temptraj

VinInn committed:
add penanly for short hits in trajFilter (tobe improved)

VinInn committed:
migrate to c++14, add index product

VinInn committed:
fix also clients

ianna committed:
* Replace legacy analyzers with one
* Cleanup

VinInn committed:
migrate to new infrastructure, part1

VinInn committed:
printouts

ianna committed:
* ME0 reco geometry payload producer
* Registration of ME0 record plugin
* Test configurations to produce and test tag locally

VinInn committed:
fixed, no regression

VinInn committed:
migrate to par desc

VinInn committed:
forgot

VinInn committed:
clean, conform

VinInn committed:
fix for hlt

VinInn committed:
seems to imporve eff and fakes

VinInn committed:
speedup, move to global

VinInn committed:
remove dict, remove stat

VinInn committed:
fix refitter

Commit:
HcalTopology depends only on HcalDDDRecConstants

VinInn committed:
fix a segfault found in a private relval

Commit:
Delete unwanted hcalTopologyConstants_cfi

Commit:
Remove all reference to hcalTopologyConstant

Commit:
Remove all reference to HcalTopology_cfi.py

VinInn committed:
fix code following review

serval2412 committed:
cppcheck: fix some reports (AnalysisAlgos)

Prefer prefix ++/-- operators for non-primitive types

serval2412 committed:
cppcheck: fix some reports
Prefer prefix ++/-- operators for non-primitive types

serval2412 committed:
cppcheck: fix some reports (CalibCalorimetry)
Prefer prefix ++/-- operators for non-primitive types

serval2412 committed:
cppcheck: fix some cppcheck reports

Prefer prefix ++/-- operators for non-primitive types

serval2412 committed:
cppcheck: fix some cppcheck reports

Prefer prefix ++/-- operators for non-primitive types

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

Commit:
Add a sub-era to configure the Stage 2 trigger

cmsbuild commented:
A new Pull Request was created by @cmsbuild  for CMSSW_8_0_X.

bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants

It involves the following packages:

CalibCalorimetry/HcalPlugins
Configuration/Geometry
Geometry/CaloEventSetup
Geometry/HcalEventSetup
Geometry/HcalTowerAlgo
HLTrigger/Configuration
L1TriggerConfig/L1ScalesProducers
SLHCUpgradeSimulations/Configuration
SimCalorimetry/HcalTrigPrimProducers


@perrotta, @cmsbuild, @civanch, @diguida, @ianna, @mdhildreth, @Martin-Grunewald, @franzoni, @Dr15Jones, @cerminar, @fwyzard, @mmusich, @mulhearn can you please review it and eventually sign? Thanks.
@ghellwig, @makortel, @geoff-smith, @jalimena, @Martin-Grunewald this is something you requested to watch as well.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@degano you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

ianna commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.
JENKINS_TEST_URL

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild commented:
-1 
 
Tested at: 43a24059e6a080d7a73ec7555a07befbcd174125
I found errors in the following addon tests: 
 
 cmsRun /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc493/cms/cmssw-patch/CMSSW_8_0_X_2015-10-18-2300/src/PhysicsTools/PatAlgos/test/IntegrationTest_cfg.py : FAILED - time: date Mon Oct 19 11:47:11 2015-date Mon Oct 19 11:47:03 2015 s - exit: 23552
cmsRun /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-18-2300/src/HLTrigger/Configuration/test/OnLine_HLT_Fake.py realData=False globalTag=@ inputFiles=@  : FAILED - time: date Mon Oct 19 12:20:42 2015-date Mon Oct 19 12:02:49 2015 s - exit: 18688
cmsRun /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-18-2300/src/HLTrigger/Configuration/test/OnLine_HLT_Fake.py realData=True  globalTag=@ inputFiles=@  : FAILED - time: date Mon Oct 19 12:06:37 2015-date Mon Oct 19 12:03:09 2015 s - exit: 18688
 
 
you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/8991/summary.html

davidlange6 committed:
Merge pull request #11931 from VinInn/TrackFitters76

Tracking Reco: tuning of outlier rejection, duplicate merging, pairSeeding

davidlange6 committed:
Merge pull request #11930 from dertexaner/HBHEIsolatedNoiseReflagger-Retuning

HBHE isolated noise reflagger retuning for Run2

davidlange6 committed:
Merge pull request #11921 from cms-tau-pog/3prong1pi0_NewDMs

3prong1pi0 new dms

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/8991/summary.html

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

ericvaandering committed:
Use perferred form of not-equal since python 3 eliminates <>

ericvaandering committed:
Python3 and recent python 2 use sys.maxsize and no sys.maxint

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

ianna commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

lgray committed:
fix up clang warnings

davidlange6 committed:
Merge pull request #11943 from heppye/patch-9

Rename Create Bd_JpsiKPi.dec to Bd_JpsiKPi.dec

cmsbuild committed:
Merge pull request #11961 from ericvaandering/fix_maxint

Python3 and recent python 2 use sys.maxsize and no sys.maxint

cmsbuild committed:
Merge pull request #11912 from dertexaner/Switch-default-HBHENoiseFilter-to-25ns-V2

Switch default HBHENoiseFilter settings to Run2-25ns configuration V2 - 76X

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild commented:
-1 
 
Tested at: 43a24059e6a080d7a73ec7555a07befbcd174125
I found errors in the following addon tests: 
 
 cmsRun /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-19-2300/src/HLTrigger/Configuration/test/OnLine_HLT_Fake.py realData=False globalTag=@ inputFiles=@  : FAILED - time: date Tue Oct 20 14:58:24 2015-date Tue Oct 20 14:48:23 2015 s - exit: 18688
cmsRun /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-19-2300/src/HLTrigger/Configuration/test/OnLine_HLT_Fake.py realData=True  globalTag=@ inputFiles=@  : FAILED - time: date Tue Oct 20 14:51:53 2015-date Tue Oct 20 14:49:01 2015 s - exit: 18688
 
 
you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9052/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9052/summary.html

ericvaandering committed:
Use new way of referring to python code

ericvaandering committed:
StandardError is now Exception

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

ericvaandering committed:
Replace has_key with in for dictionaries

ericvaandering committed:
xreadlines is obsolete

cmsbuild committed:
Merge pull request #11983 from lgray/geometry_clang_warnings

Fix clang warnings in geometry classes.

cmsbuild committed:
Merge pull request #11918 from igv4321/HcalSimpleReconstructor-with-method3

Hcal simple reconstructor with method3

Martin-Grunewald commented:
-1
No direct modification of HLT menu dumps

ericvaandering committed:
reduce() is not a builtin in python 3

ericvaandering committed:
backticks are deprecated, use repr() instead

ericvaandering committed:
types module is deprecated

ericvaandering committed:
Make sure output of zip is a list (is a generator in python 3

bsunanda commented:
@Martin-Grunewald I am deleting some files used in these test python files. How would you guarantee the unit tests of HLT's then if I do not edit these files explicitly
________________________________
From: Martin Grunewald [notifications@github.com]
Sent: 21 October 2015 09:43
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants (#11920)


-1
No direct modification of HLT menu dumps

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11920#issuecomment-149806615>.

ericvaandering committed:
Use next() built-in

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

Martin-Grunewald commented:
Put the deletion of the PSet as a python command into 
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/HLTrigger/Configuration/python/customizeHLTforCMSSW.py

See in that file in how to organise that part!

Commit:
Add a Run2_2016 top-level era that uses the stage 2 trigger

Commit:
Add a Run1 era which is currently empty

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

lgray committed:
fix failing relval missing new ECAL digi params

Dr15Jones committed:
Set ROOT histogram fitting to be thread-safe

In order to fit different root histograms on different threads one
must call the undocumented function TMinuitMinimizer::UseStaticMinuit
and pass it false.
This will fix crashes seen in the CMSSW_7_6 IBs.

Dr15Jones committed:
Removed LazyGetter from SiPixelClusterShapeCache

The original LazyGetter implementation was not thread-safe and the
facility was never used in production. Removing the code avoids
accidently trying to switch on that facility.
The SiPixelClusterShapeCacheProducer was also switch from a stream
producer to a global producer.

cmsbuild committed:
Merge pull request #12035 from Dr15Jones/removeLazyGetterFromSiPixelClusterShapeCache

Removed LazyGetter from SiPixelClusterShapeCache

davidlange6 committed:
Merge pull request #11997 from ericvaandering/fix_standarderror

StandardError is now Exception

cmsbuild committed:
Merge pull request #12026 from nickmccoll/StaticAnalyzerErrors_RecoLocalTracker

Removed static analyzer errors from RecoLocalTracker

cmsbuild committed:
Merge pull request #12011 from ericvaandering/fix_types

types module is deprecated

cmsbuild committed:
Merge pull request #12005 from ericvaandering/fix_xreadlines

xreadlines is obsolete

cmsbuild committed:
Merge pull request #11924 from dertexaner/Add-NEF-to-HcalRecHitFlagsToBeExcluded

Adding HBHENegativeNoise bit to HcalRecHitFlagsToBeExcluded - 76X

cmsbuild committed:
Merge pull request #11899 from capalmer85/update-pcc-76X-oct15

PCC ntupler 76x - October2015

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

davidlange6 committed:
Merge pull request #12027 from lgray/fix_ecaldigitizer_datamix

Fix failing relval missing new ECAL digi params (80X)

cmsbuild committed:
Merge pull request #11916 from lveldere/fastsim-hf-sl-run1-take2

Fastsim: HF: use shower library also for run1

davidlange6 committed:
Merge pull request #11884 from nhanvtran/puppiV9-76x-pass2

update to PUPPI;speedup, low PU corr

cmsbuild committed:
Merge pull request #11875 from rappoccio/rappoccio_76x_gridrhoNoHF

Adding no-HF grid-based rho : 76x forward-port #11773

davidlange6 committed:
Merge pull request #12004 from ericvaandering/fix_haskey

Replace has_key with in for dictionaries

davidlange6 committed:
Merge pull request #11960 from ericvaandering/fix_ne

Use preferred form of not-equal since python 3 eliminates <>

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

hengne committed:
fix typo in Era specification

hengne committed:
rebase to head -- Merge branch 'CMSSW_8_0_X' into from-CMSSW_8_0_X_2015-10-22-2300_fixEraTypo

davidlange6 committed:
protect against bogus hcal det Ids even if they should not exist

davidlange6 committed:
whoops its when its NOT valid

Commit:
Remove conflicts

gennai committed:
Added a parameter in the Harvester to avoid considering LS which have a too small sigmaZ values. Default cut is at 1 cm

gennai committed:
Added a parameter in the Harvester to avoid considering LS which have a too small sigmaZ values. Default cut is at 1 cm

gennai committed:
Added a parameter in the Harvester to avoid considering LS which have a too small sigmaZ values. Default cut is at 1 cm

gennai committed:
Added a parameter in the Harvester to avoid considering LS which have a too small sigmaZ values. Default cut is at 1 cm

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

davidlange6 committed:
Merge pull request #12066 from davidlange6/hcalDetIdProtect

Hcal det id protect again invalid IDs in digitizer

davidlange6 committed:
Merge pull request #12017 from ericvaandering/fix_next

Use next() built-in

davidlange6 committed:
Merge pull request #12065 from hengne/from-CMSSW_8_0_X_2015-10-22-2300_fixEraTypo

fix a typo in --era specification

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

Martin-Grunewald commented:
This PR contains merge conflicts and claims to have to change 3565 files....
I suggest to rebase.

cmsbuild commented:
Pull request #11920 was updated. @smuzaffar, @diguida, @cerminar, @cmsbuild, @franzoni, @Dr15Jones, @mmusich, @mulhearn can you please check and sign again.

Commit:
Resolve conflict

bsunanda commented:
@cmsbuild Please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
Pull request #11920 was updated. @diguida, @lveldere, @cmsdoxy, @cerminar, @Martin-Grunewald, @covarell, @bendavid, @vanbesien, @govoni, @perrotta, @civanch, @cmsbuild, @fwyzard, @davidlange6, @vciulli, @smuzaffar, @Dr15Jones, @cvuosalo, @mdhildreth, @ianna, @deguio, @slava77, @ggovi, @vadler, @mmusich, @mulhearn, @ssekmen, @danduggan, @thuer, @monttj, @franzoni can you please check and sign again.

cmsbuild commented:
+1
Tested at: 987644c65e0a765431876ea063e7c24e2ad00f68
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9175/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/987644c65e0a765431876ea063e7c24e2ad00f68
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9175/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9175/git-merge-result

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9175/summary.html

civanch commented:
+1

cmsbuild committed:
Merge pull request #12077 from gennai/TkAlcaRecoProducers_FIXES_V2_CMSSW_8_0_X

Tk alca reco producers fixes v2 cmssw 8 0 x

davidlange6 commented:
@bsunanda - seems this can use a further rebase.. which are the actual commits you want to request?

cmsbuild committed:
Merge pull request #12032 from Dr15Jones/makeHistFitThreadSafe_8_0

Set ROOT histogram fitting to be thread-safe

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

davidlange6 committed:
Merge pull request #12012 from ericvaandering/fix_zip

Make sure output of zip is a list (is a generator in python 3

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

slava77 commented:
-1
merge conflicts should be resolved

davidlange6 committed:
Merge pull request #12007 from ericvaandering/fix_reduce

reduce() is not a builtin in python 3

davidlange6 committed:
Merge pull request #12023 from mark-grimes/add2016AndRun1Eras

Add eras for 2016 and Run 1

cmsbuild committed:
Merge pull request #11935 from ianna/me0-reco-ideal-geometry-db-record

ME0 DB Record

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

bsunanda commented:
@davidlange6 I believe this is for CMSSW version 8_0_0? Please confirm
________________________________
From: Slava Krutelyov [notifications@github.com]
Sent: 26 October 2015 13:32
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants (#11920)


-1
merge conflicts should be resolved

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11920#issuecomment-151119549>.

davidlange6 commented:
yes, it is requested for the 80x release milestone.

> On Oct 26, 2015, at 2:51 PM, bsunanda <notifications@github.com> wrote:
> 
> @davidlange6 I believe this is for CMSSW version 8_0_0? Please confirm
> ________________________________
> From: Slava Krutelyov [notifications@github.com]
> Sent: 26 October 2015 13:32
> To: cms-sw/cmssw
> Cc: Sunanda Banerjee
> Subject: Re: [cmssw] bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants (#11920)
> 
> 
> -1
> merge conflicts should be resolved
> 
> —
> Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11920#issuecomment-151119549>.
> —
> Reply to this email directly or view it on GitHub.
>

Commit:
Solve the merge conflict issues

davidlange6 committed:
Merge pull request #11996 from ericvaandering/fix_attrs

Use new way of referring to python code

bsunanda commented:
Done. Pushed the changes
________________________________
From: David Lange [notifications@github.com]
Sent: 26 October 2015 14:59
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants (#11920)

yes, it is requested for the 80x release milestone.

> On Oct 26, 2015, at 2:51 PM, bsunanda <notifications@github.com> wrote:
>
> @davidlange6 I believe this is for CMSSW version 8_0_0? Please confirm
> ________________________________
> From: Slava Krutelyov [notifications@github.com]
> Sent: 26 October 2015 13:32
> To: cms-sw/cmssw
> Cc: Sunanda Banerjee
> Subject: Re: [cmssw] bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants (#11920)
>
>
> -1
> merge conflicts should be resolved
>
> —
> Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11920#issuecomment-151119549>.
> —
> Reply to this email directly or view it on GitHub.
>


—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11920#issuecomment-151141969>.

davidlange6 committed:
Merge pull request #11874 from serval2412/CMSSW_7_6_X

cppcheck: some fixes for 4 modules

cmsbuild commented:
Pull request #11920 was updated. @diguida, @lveldere, @cmsdoxy, @cerminar, @Martin-Grunewald, @covarell, @bendavid, @vanbesien, @govoni, @perrotta, @civanch, @cmsbuild, @fwyzard, @davidlange6, @vciulli, @smuzaffar, @Dr15Jones, @cvuosalo, @mdhildreth, @ianna, @deguio, @slava77, @ggovi, @vadler, @mmusich, @mulhearn, @ssekmen, @danduggan, @thuer, @monttj, @franzoni can you please check and sign again.

bsunanda commented:
@cmsbuild Please test
________________________________
From: cmsbuild [notifications@github.com]
Sent: 26 October 2015 15:17
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants (#11920)


Pull request #11920<https://github.com/cms-sw/cmssw/pull/11920> was updated. @diguida<https://github.com/diguida>, @lveldere<https://github.com/lveldere>, @cmsdoxy<https://github.com/cmsdoxy>, @cerminar<https://github.com/cerminar>, @Martin-Grunewald<https://github.com/Martin-Grunewald>, @covarell<https://github.com/covarell>, @bendavid<https://github.com/bendavid>, @vanbesien<https://github.com/vanbesien>, @govoni<https://github.com/govoni>, @perrotta<https://github.com/perrotta>, @ civanch<https://github.com/civanch>, @cmsbuild<https://github.com/cmsbuild>, @fwyzard<https://github.com/fwyzard>, @davidlange6<https://github.com/davidlange6>, @vciulli<https://github.com/vciulli>, @smuzaffar<https://github.com/smuzaffar>, @Dr15Jones<https://github.com/Dr15Jones>, @cvuosalo<https://github.com/cvuosalo>, @mdhildreth<https://github.com/mdhildreth>, @ianna<https://github.com/ianna>, @deguio<https://github.com/deguio>, @slava77<https://github.com/slava77>, @ggovi<https://github.com/ggovi>, @vadler<https://github.com/vadler>, @mmusich<UrlBlockedError.aspx>, @mulhearn<https://github.com/mulhearn>, @ssekmen<https://github.com/ssekmen>, @danduggan<https://github.com/danduggan>, @thuer<https://github.com/thuer>, @monttj<https://github.com/monttj>, @franzoni<https://github.com/franzoni> can you please check and sign again.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11920#issuecomment-151149513>.

davidlange6 committed:
Merge pull request #12008 from ericvaandering/fix_repr

backticks are deprecated, use repr() instead

davidlange6 commented:
it doesn't seem that the PR has improved much.. it merges now but has 100+ commits. What procedure are you following ? (or do you expect 100 commits to be newly included)

cmsbuild commented:
The tests are being triggered in jenkins.

Martin-Grunewald commented on HLTrigger/Configuration/python/customizeHLTforCMSSW.py:
```diff
@@ -203,6 +216,7 @@ def customiseHLTforCMSSW(process, menuType="GRun", fastSim=False):
         process = customiseFor11183(process)
         process = customiseFor11497(process)
         process = customiseFor12044(process)
+        process = customiseFor11920(process)
```
This should be under a new if statement ```if cmsswVersion >= "CMSSW_8_0":``` as it seems this will not be put in 76X.

bsunanda commented:
@davidlange6 i take the latest IB. I use cos-merge-topic to merge the branch. For conflicts I edit the files and do a git commit and then do a git push. Can you suggest a better method to avoid the history
________________________________
From: David Lange [notifications@github.com]
Sent: 26 October 2015 15:18
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx41 HcalTopology depends only on HcalDDDRecConstants (#11920)


it doesn't seem that the PR has improved much.. it merges now but has 100+ commits. What procedure are you following ? (or do you expect 100 commits to be newly included)

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11920#issuecomment-151150995>.

cmsbuild committed:
Merge CMSSW_7_6_X into CMSSW_8_0_X.

cmsbuild commented:
+1
Tested at: 4d592caae5487af98313ea5686cf04b7a2143ada
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9200/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/4d592caae5487af98313ea5686cf04b7a2143ada
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9200/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9200/git-merge-result

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9200/summary.html

slava77 commented:
-1
rebase needed

Martin-Grunewald commented:
-1
see HLT issue above

Commit:
Merged Run2-hcx41 from repository bsunanda

Commit:
Take care of comment from Martin

ianna commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
Pull request #11920 was updated. @covarell, @lveldere, @cmsdoxy, @cerminar, @fabozzi, @Martin-Grunewald, @diguida, @bendavid, @vanbesien, @govoni, @perrotta, @civanch, @franzoni, @cmsbuild, @fwyzard, @davidlange6, @vciulli, @smuzaffar, @Dr15Jones, @cvuosalo, @mdhildreth, @ianna, @deguio, @slava77, @ggovi, @vadler, @mmusich, @mulhearn, @ssekmen, @danduggan, @boudoul, @thuer, @monttj, @srimanob can you please check and sign again.

cmsbuild commented:
+1
Tested at: fb7ebdc87da1eef489a6d6859608f8f410e294cc
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9238/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/c03c0aa6bc17092e47a7f4e30ab91b3a31139fd8
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9238/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9238/git-merge-result

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11920/9238/summary.html

slava77 commented:
-1
merge conflicts and need for rebase

bsunanda commented:
@davidlange6 Please close this PR because #12196 is approved to be integrated
